Source/WebKit/WebProcess/WebStorage/StorageNamespaceImpl.cpp

```diff
@@ -42,19 +42,9 @@
 namespace WebKit {
 using namespace WebCore;
 
-static HashMap<StorageNamespaceImpl::Identifier, StorageNamespaceImpl*>& sessionStorageNamespaces()
+Ref<StorageNamespaceImpl> StorageNamespaceImpl::createSessionStorageNamespace(Identifier identifier, PageIdentifier pageID, WebCore::SecurityOrigin& topLevelOrigin, unsigned quotaInBytes)
```

Comment:
Can we pass the topLevelOrigin as a `const WebCore::SecurityOrigin&` ?

Reply:
Done.